### DISCUSSION TITLE
How can i convert the component to HTML in nodejs ?

### DISCUSSION THREAD

**Discussion by issam-seghir** [resend/react-email#1411] [Q&A]
in the example from the [documentation](https://github.com/resend/react-email/tree/main/examples/nodemailer)  the  component and the code that converts the component to HTML are in the same folder. 
However, in my case, I have separate repositories for my frontend and backend , how can I convert the component to HTML in this scenario in my server code?

**Comment by gabrielmfern** [comment]
So, the example is made to be the code you can use in the server. It just adds a babel as pre-processor to support the `.jsx`/`.tsx` file extension. Even though it is using React, it will run fine as it only uses React's server APIs which are exactly made to run under Node.

**Reply by issam-seghir** [reply]
Thank you for the response, but I mean how can i access the react component in my server code? do i need a monorepo setup to use the package?

**Reply by gabrielmfern** [reply]
You don't need a monorepo setup. You just need to import the component, and as long as you have the right preprocessor, things will work as you would expect when using Next or any other thing. No caveats in that case.
You can do this with a monorepo setup, but it's going to be a bit more of a headache if you preprocess one thing and not the other.

**Comment by ryanbitonio** [comment]
Any update on this? I'm also looking for an answer to this.
